wallet/wallet.go

```diff
@@ -498,6 +498,11 @@ out:
 				continue
 			}
 			timeout = req.lockAfter
+			if timeout == nil {
+				log.Info("Wallet unlocked without timeout")
+			} else {
+				log.Info("Wallet temporarily unlocked")
```

Comment:
I edited my comment above with why logging the duration would not be a good idea.